### ISSUE TITLE
java.lang.RuntimeException: missing OpenAPI input! when tryin to build the package 

### ISSUE THREAD

**Issue by elisabetao** [AbsaOSS/spline-spark-agent#185]
Hi,

When trying to build the agent  I am hitting the following error:
```
[INFO] --- swagger-codegen-maven-plugin:3.0.23:generate (swagger.api.v1_0) @ agent-core_2.12 ---
[WARNING] Output directory does not exist, or is inaccessible. No file (.swagger-codegen-ignore) will be evaluated.
[ERROR]
java.lang.RuntimeException: missing OpenAPI input!
    at io.swagger.codegen.v3.DefaultGenerator.generate (DefaultGenerator.java:768)
```
Anyone has encountered this/has any ideas?

Thanks a lot

**Comment by wajda** [comment]
That apparently is a Windows related issue, we're investigating.
We run builds primarily on Linux or MacOS, there we never had such issue.

**Comment by wajda** [comment]
fyi @oleksandr-korchak

**Comment by cerveada** [comment]
This is caused by swagger-codegen bug:
https://github.com/swagger-api/swagger-codegen/issues/10844

We will fix it by downgrading to version 3.0.21 where the bug is not present for now.
